pom.xml

```diff
@@ -425,6 +437,18 @@
             <version>64.2</version>
         </dependency>
 
+        <!-- Benchmark -->
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-core</artifactId>
+            <version>1.19</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-generator-annprocess</artifactId>
+            <version>1.19</version>
+        </dependency>
+
```

Comment:
Make sure these are not included in the final PR please

Reply:
I want to talk about this. I think keeping the bench mark is a good idea in case we make changes in the future and want to see if they work better/worse (like switch providers)

Reply:
It would be a good idea to keep it but probably not in the main POM. Maybe a maven submodule